server/utils/applications/utils.ts

```diff
@@ -379,14 +367,33 @@ const lengthOfStayForUI = (duration: number) => {
 
 export const applicationShowPageTabs = {
   application: 'application',
+  assessment: 'assessment',
```

Comment:
I'm not sure this enum replacement is terribly useful -- it might work better to simply have the map of tab => labels and enforce that the tab is of type `ApplicationShowPageTab` in the controller?